src/user_actions_handler.py

```diff
@@ -77,5 +166,10 @@ def get_handler(operator):
     "exit": handler_bye,
     "good bye": handler_bye,
     "sort dir": handler_sort,
-    "add contact": handler_add_contact
+    "show all contacts": show_all_contacts,
```

Comment:
update all naming for hadlers based in template:
```
handler_bye
handler_sort
handler_add_contact
```

Reply:
Ok